src/main/java/duke/ToDo.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+
+/**
+ * Represents a To Do with a specified description, and boolean to indicate if completed or not.
+ */
+public class ToDo extends Task {
+    public ToDo(String description) {
+        super(description);
+    }
+
+    public String recordString() {
```

Comment:
Perhaps it will be good to add the `@Override` tag here? (So that compiler can check if overriding is done properly!)